app/javascript/elm/tests/MainTests.elm

```diff
@@ -525,132 +528,173 @@ sessionWithTitle title =
+    , id = 123
+    }
+
+
+selectedSessionWithId : Int -> SelectedSession
+selectedSessionWithId id =
+    { selectedSession | id = id }
+
+
 viewTests : Test
 viewTests =
     describe "view"
-        [ fuzz2 string string "session titles are displayed in the list" <|
+        [ fuzz2 string string "with no selection session titles are displayed in the list" <|
             \title1 title2 ->
-                { defaultModel | sessions = [ sessionWithTitle title1, sessionWithTitle title2 ], selectedSessionId = Nothing }
+                { defaultModel | sessions = [ sessionWithTitle title1, sessionWithTitle title2 ], selectedSession = NotAsked }
                     |> view
                     |> Query.fromHtml
                     |> Query.contains
                         [ Html.text title1
                         , Html.text title2
                         ]
-        , fuzz (intRange 1 10) "with modulo 50 sessions in the model the load more button is shown" <|
+        , fuzz (intRange 1 10) "with no selection and modulo 50 sessions in the model the load more button is shown" <|
             \times ->
-                { defaultModel | sessions = List.repeat (50 * times) session, selectedSessionId = Nothing }
+                { defaultModel | sessions = List.repeat (50 * times) session, selectedSession = NotAsked }
                     |> view
                     |> Query.fromHtml
                     |> Query.contains [ Html.text "Load More..." ]
-        , test "with 0 sessions in the model the load more button is not shown" <|
+        , test "with no selection and 0 sessions in the model the load more button is not shown" <|
             \times ->
-                { defaultModel | sessions = [], selectedSessionId = Nothing }
+                { defaultModel | sessions = [], selectedSession = NotAsked }
                     |> view
                     |> Query.fromHtml
                     |> Query.findAll [ Slc.text "Load More..." ]
                     |> Query.count (Expect.equal 0)
-        , fuzz int "with 1 sessions in the model the export link is correctly generated" <|
+        , fuzz int "with no selection and 1 sessions in the model the export link is correctly generated" <|
             \id ->
                 let
                     expected =
                         exportPath ++ "?session_ids[]=" ++ String.fromInt id
                 in
-                { defaultModel | sessions = [ sessionWithId id ] }
+                { defaultModel | sessions = [ sessionWithId id ], selectedSession = NotAsked }
                     |> view
                     |> Query.fromHtml
                     |> Query.find [ Slc.containing [ Slc.text "export sessions" ] ]
                     |> Query.has [ Slc.attribute <| Attr.href expected ]
-        , fuzz int "with 2 sessions in the model the export link is correctly generated" <|
+        , fuzz int "with no selection and 2 sessions in the model the export link is correctly generated" <|
```

Comment:
is this test needed? or the one with 1 session?

Reply:
I guess it's a bit of overtesting. The rationale was that in one case there's `?` and stuff and in the other `?...&`. I'm fine with removing if you prefer